CONTRIBUTING.md

```diff
@@ -1,171 +1,28 @@
-or any other Google open-source project, you shouldn't have to do so again. You
-can review your signed CLAs at
-[cla.developers.google.com/clas](https://cla.developers.google.com/clas).
-
-First, check that you are signed in to a [Google
-Account](https://accounts.google.com) that matches your [local Git email
-address](https://help.github.com/articles/setting-your-email-in-git/). Then
-choose one of the following:
-
-* If you are **an individual writing original source code** and **you own the
-  intellectual property**, sign the [Individual
-  CLA](https://developers.google.com/open-source/cla/individual).
-* If you work for **a company that wants to allow you to contribute**, sign the
-  [Corporate CLA](https://developers.google.com/open-source/cla/corporate).
-
-You (and your authorized signer, if corporate) can sign the CLA
-electronically. After that, we'll be able to accept your contributions.
-
-## Contributor Code of Conduct
-
-As contributors and maintainers of this project, and in the interest of
-fostering an open and welcoming community, we pledge to respect all people who
-contribute through reporting issues, posting feature requests, updating
-documentation, submitting pull requests or patches, and other activities.
-
-We are committed to making participation in this project a harassment-free
-experience for everyone, regardless of level of experience, gender, gender
-identity and expression, sexual orientation, disability, personal appearance,
-body size, race, ethnicity, age, religion, or nationality.
-
-Examples of unacceptable behavior by participants include:
-
-* The use of sexualized language or imagery
-* Personal attacks
-* Trolling or insulting/derogatory comments
-* Public or private harassment
-* Publishing other's private information, such as physical or electronic
-addresses, without explicit permission
-* Other unethical or unprofessional conduct.
-
-Project maintainers have the right and responsibility to remove, edit, or reject
-comments, commits, code, wiki edits, issues, and other contributions that are
-not aligned to this Code of Conduct.  By adopting this Code of Conduct, project
-maintainers commit themselves to fairly and consistently applying these
-principles to every aspect of managing this project.  Project maintainers who do
-not follow or enforce the Code of Conduct may be permanently removed from the
-project team.
-
-This code of conduct applies both within project spaces and in public spaces
-when an individual is representing the project or its community.
-
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by opening an issue or contacting one or more of the project
-maintainers.
-
-This Code of Conduct is adapted from the [Contributor
-Covenant](http://contributor-covenant.org), version 1.2.0, available at
-[http://contributor-covenant.org/version/1/2/0/](http://contributor-covenant.org/version/1/2/0/)
-
-### Running tests
```

Comment:
Should we keep this section?

Reply:
I've created a more thoughful merge of CONTRIBUTING.md.